app/src/main/java/com/checkin/app/checkin/home/epoxy/CityLocationModelHolder.kt

```diff
@@ -0,0 +1,42 @@
+package com.checkin.app.checkin.home.epoxy
+
+import android.view.View
+import android.widget.TextView
+import butterknife.BindView
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import com.airbnb.epoxy.EpoxyModelWithHolder
+import com.checkin.app.checkin.R
+import com.checkin.app.checkin.home.model.CityLocationModel
+import com.checkin.app.checkin.misc.epoxy.BaseEpoxyHolder
+
+@EpoxyModelClass(layout = R.layout.item_user_location_city)
+abstract class CityLocationModelHolder : EpoxyModelWithHolder<CityLocationModelHolder.Holder>() {
+
+    @EpoxyAttribute
+    internal lateinit var data: CityLocationModel
+
+    @EpoxyAttribute
+    internal lateinit var cityListener: View.OnClickListener
```

Comment:
You should not use view click listener, but rather create a new interface. That's the standard way for the communication between different modules.